src/patternfly/demos/Alert/examples/alert-toast-example.hbs

```diff
@@ -1,5 +1,5 @@
 {{> page-demo-default}}
-{{#> alert-group alert-group--attribute='role="status"' alert-group--modifier="pf-m-toast"}}
+{{#> alert-group alert-group--modifier="pf-m-toast"}}
```

Comment:
So weird! I see that you removed role="status", but it's still showing up in this example:

```
<ul class="pf-c-alert-group pf-m-toast" role="status">
  <li class="pf-c-alert-group__item">
```